.changeset/curly-crabs-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-circleci': patch
+---
+
+Migrated to new composability API, exporting the plugin instance as `circleCIPlugin`, the entity page content as `EntityCircleCIContent`, and entity conditional as `isCircleCIAvailable`.
```

Comment:
It's because of the existing `circleCIApiRef` and `CircleCIApi`. Agree that `circleciPlugin` is prolly a bit nicer, although that's pretty ugly too tbh, but overall not worth either diverging or renaming the existing symbols.